aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -202,6 +207,95 @@ For vertical lists:
+
+<Example
+  docs="https://v2.grommet.io/namevaluelist?theme=hpe"
+  height={{ min: 'small' }}
+  code={[
+  "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListDetailPage.js",
+  "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/data.js",
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListDetailPage.js"
+  template
+  showResponsiveControls={false}
+  width="xlarge"
+  pad="large"
+>
+  <NameValueListDetailPage />
+</Example>
+
+### When To Omit “Empty value” Name-Value Pairs
+
+In certain scenarios, it is not necessary to display name-value pairs that don’t have a value. These scenarios include:
+
+**Public-facing Profiles** - For instances where a profile is outwardly facing, only name-value pairs with values should be displayed. The complete list of possible entries should only be viewable when a user goes to edit their profile.
+
+<Example
+  docs="https://v2.grommet.io/namevaluelist?theme=hpe#layout"
+  code={["https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/data.js",
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js"
+  height={{ min: 'small' }}
+  pad="large"
+  showResponsiveControls={false}
+  template
+>
+  <NameValueListEditExample />
+</Example>
+
+**Summaries (such as Dashboard Cards)** - Summaries are intended to display information-salient content, often in areas with restricted screen real estate. Given this, name-value pairs without a value should be omitted.
+
+<Example
+  docs="https://v2.grommet.io/namevaluelist?theme=hpe#layout"
+  code={["https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListSummaryExample.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/data.js",
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListSummaryExample.js"
+  height={{ min: 'small' }}
+  pad="large"
+  showResponsiveControls={false}
+  template
+>
+  <NameValueListSummaryExample />
+</Example>
+
+### Accessibility for Name-Value Pairs with Empty Value
+
+Add an `a11yTitle="No data"` to the Text component that would otherwise render the data to ensure users of assisstive technology are informed of the empty state.
```

Comment:
This feels a bit buried to me. I wonder if we should pull this up to L213. Making a comment/suggestion there.